src/custom-elements/src/components/button/button.scss

```diff
@@ -0,0 +1,8 @@
+:host(chi-button) {
+  @import '../../global/styles/common';
+  @import 'components/buttons/_button-groups-variables';
+  @import 'components/buttons/_horizontal-button-groups';
+  @import 'components/buttons/_vertical-button-groups';
+  @import 'components/buttons/button-groups';
+  @import 'components/buttons/buttons';
+}
```

Comment:
Add display: inline-flex; 
See comments below regarding this change.  
This display property should be also applied to src/custom-elements/src/components/icon/icon.scss

Reply:
> Add display: inline-flex;
> See comments below regarding this change.
> This display property should be also applied to src/custom-elements/src/components/icon/icon.scss

@srinath-koneru Review this comment, please.